docker/api/coverage.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+cd /opt/cnaas/venv/cnaas-nms/src/
+
+source ../../bin/activate
+
+ls -l .coverage-*
+coverage combine .coverage-*
+coverage report --omit='*/site-packages/*'
+coverage xml -i --omit='*/site-packages/*,*/templates/*'
+
+export CODECOV_TOKEN="dbe13a97-70b5-49df-865e-d9b58c4e9742"
+bash <(curl -s https://codecov.io/bash)
```

Comment:
I'm still thinking this should be replaced with a separate github action step - keeping the CODECOV_TOKEN in the clear like this isn't a good idea. And I'm not sure I see the reason why a codecov upload outside the context of GitHub pipelines is interesting anyway (why would I upload the codecov result in a manual test run on my machine? Is there some usecase of codecov.io I'm not aware of?)

Reply:
Full docs here: https://github.com/codecov/codecov-action

Reply:
Since we have a public repo I guess we don't even need to use a token and deal with this secret not available to forks etc then. But still need to get the coverage file out from the container so the github actions host can read it. Anyway I think it can be a separate PR